src/pages/kimseulki/Login/Login.js

```diff
@@ -0,0 +1,55 @@
+import React, { useState } from "react";
+import "./Login.scss";
+import "../../../style/reset.scss";
+import { Link } from 'react-router-dom';
+import { useNavigate } from 'react-router-dom'; 
+
+const Login = () => {
+  const [id, setId] = useState('');
+  const [pw, setPw] = useState('');
+  const [isValid, setIsValid] = useState(false);
+
+  const navigate = useNavigate();
+
+
+  const saveUserId = (event) => {
+    setId(event.target.value);
+  };
+
+  const saveUserPw = (event) => {
+    setPw(event.target.value);
+  };
+
+  const loginValidation = () => {
+    return id.includes("@") && pw.length >= 5
+      ? setIsValid(true)
+      : setIsValid(false);
+  };
+
+  const goToMain = () => {
+    navigate('/kim-main')
+  }
+
+  return (
+    <div className="login">
+      <h1 className="title_hd">로그인페이지</h1>
+       <div className="login_content">
+        <div className="login_logo">
+          <p className="login_font logo_font">westagram</p>
+        </div>
+        <div className="login_inputWrap">
+          <input type="text" placeholder="전화번호, 사용자 이름 또는 이메일" id="id"
+            value={id} onChange={saveUserId} onKeyUp={loginValidation}/>
+          <input type="password" placeholder="비밀번호" id="pw"
+            value={pw} onChange={saveUserPw} onKeyUp={loginValidation}></input>
```

Comment:
한가지 input에 onChange와 onKeyUp과 같은 비슷한 동작을 인식하는 함수는 한가지만 작동할 수 있게 하는것이 예상치 못한 에러를 방지할 수 있는데에 효과적입니다.
```suggestion
          <input type="text" placeholder="전화번호, 사용자 이름 또는 이메일" id="id"
            value={id} onChange={saveUserId}/>
          <input type="password" placeholder="비밀번호" id="pw"
            value={pw} onChange={saveUserPw} ></input>
```
loginValidation이 반환하는 값을 함수가 아닌 방식으로 컨트롤할 수 있는 방법을 고민해보세요!